### ISSUE TITLE
Problems in Running 'record_parser_save.py'

### ISSUE THREAD

**Issue by GabrielZZZ** [ApolloAuto/apollo#13022]
Hi, I want to extract pcd files from a record, and I am following the step listed here [Apollo cyber-record file parser (Python based)](https://github.com/ApolloAuto/apollo/tree/3711be4317b62fffa548f91cb383269d4cc25987/modules/tools/record_parse_save]). The error message is that `from modules.drivers.proto.pointcloud_pb2 import PointCloud
ModuleNotFoundError: No module named 'modules.drivers.proto.pointcloud_pb2'.` I have looked through the code, and found that there is no `pointcloud_pb2.py` but `point_cloud.proto` in `modules/drivers/proto`.  
![Screenshot from 2020-11-17 21-14-44](https://user-images.githubusercontent.com/42797233/99396029-35996300-291c-11eb-89d5-37d4849c4b42.png)

Maybe I need to compile the protocol buffers manually but I am not sure, since I think it should be done during the build process of Apollo. Can you take a look at it and offer me some tips? Thanks!

**Comment by daohu527** [comment]
You can try below cmd
```
bazel build modules/tools/record_parse_save:record_parse_save
```

then run the cmd
```
./bazel-bin/modules/tools/record_parse_save/record_parse_save
```

## python
It seems that running python code must first generate proto, and the script for generating proto has been deleted in 6.0.

**Comment by GabrielZZZ** [comment]
Thanks @daohu527 . The first build cmd was successful, but the second cmd I got the following error:
![Screenshot from 2020-11-18 12-20-54](https://user-images.githubusercontent.com/42797233/99483157-9f575280-2998-11eb-80fe-ffd6941d2287.png)

**Comment by changsh726** [comment]
Thanks for your reporting! This has been fixed in #13029, and instructions have also been updated. Please pull the latest master and have a try.

**Comment by GabrielZZZ** [comment]
Thanks!
